.github/workflows/pr-builder.yml

```diff
@@ -17,16 +17,20 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    strategy:
```

Comment:
What is the purpose of including this if is not being utilized in the `java-version`

Reply:
I've already opened https://github.com/wso2/identity-tools/pull/51 to fix this.